### ISSUE TITLE
Show running order of talks at a conference

### ISSUE THREAD

**Issue by marcoroth** [adrienpoly/rubyvideo#181]
We currently don't have this data in the static YAML files, but the last few conferences I added I made sure that they are in order and already grouped by the right conference day.

Maybe we can also make this gradual and only show it for conference we have the data, so we don't have to do it all at once.

I really like this feature because a lot of times at conferences you hear somebody say "as we heard earlier in XY's talk" in a talk and this makes it super simple to see which talk they could've meant.

Also, for some conferences it really makes a difference to see if a talk happened on the first or the last day.

![CleanShot 2024-09-09 at 12 36 07](https://github.com/user-attachments/assets/3e7ab1ff-fc6a-4d16-84d2-a5734f7760d7)

**Comment by adrienpoly** [comment]
Yes this is a nice suggestion. Initially I wanted to show related talks in this section (exploratory work was done here #19) but it could make send to display the list of talks from this event and maybe get a tiny tab where we would lazy load recommended talks

**Comment by marcoroth** [comment]
Maybe we can still do both. YouTube also shows related talks under the "playlist".

![CleanShot 2024-09-09 at 13 14 30](https://github.com/user-attachments/assets/251ae83d-4f30-434d-bd91-f265aae97faa)
